### ISSUE TITLE
Core glTF 2.0 support

### ISSUE THREAD

**Issue by vorg** [pex-gl/pex-renderer#147]
Currently there is few examples for loading glTF each using copy pasted piece of code. As we are aligning our features with glTF it would be good to have glTF support in the core.

There are different approaches

Scene loader:
```javascript
const scene = await rederer.loadGltf('assets/damaged_helmet/damaged_helmet.gltf')
renderer.add(scene.root)
```

Scene component:
```javascript
const gltfEnt = renderer.entity([
  rederer.gltf({ src: 'assets/damaged_helmet/damaged_helmet.gltf' })
])
renderer.add(gltfEnt)
```

Interesting extensions
- [EXT_lights_image_based](https://github.com/KhronosGroup/glTF/pull/1480)

**Comment by dmnsgn** [comment]
I'd go for a modified option 1: scene loader.
It would be good to keep loading and parsing decoupled IMO.
The glTF loader should handle the binaries and images.
The glTF parser should be responsible for creating pex-renderer entities.

This way we can still keep the freedom of updating a scene/model's properties/images at runtime before rendering it.

To sum up, I don't see the glTF functionality as a component but as an entity builder.

**Comment by dmnsgn** [comment]
Fixed by dcc40ef8081c621caaba9487e35ebfbbded48100
